README.md

```diff
@@ -1,46 +1,63 @@
 [![Code-samples-compile](https://github.com/iolivia/rust-sokoban/workflows/Code-samples-compile/badge.svg)](https://github.com/iolivia/rust-sokoban/actions?query=workflow%3ACode-samples-compile)
-[简体中文](README-zh_CN.md)
-# [Rust sokoban](https://sokoban.iolivia.me)
 
-## :dart: What is it?
-This repository hosts the Rust Sokoban book and source code used in the book. It's using [mdbook](https://github.com/rust-lang/mdBook). You can read the book at [sokoban.iolivia.me](https://sokoban.iolivia.me). 
+[English](README-en.md)
+
+# [Rust编写推箱子游戏教程 ](https://sokoban.iolivia.me)
+
+## :dart: 这是什么?
+这里是使用[mdbook](https://github.com/rust-lang/mdBook)创建的《Rust编写推箱子游戏》教程和相关代码。想在线阅读下？ 好嘞，给您安排，点这里[sokoban.iolivia.me](https://sokoban.iolivia.me)。
```

Comment:
can we keep the main README in english please and keep the link to the zh_CN one?